compiler/rustc_lint/messages.ftl

```diff
@@ -401,6 +397,19 @@ lint_incomplete_include =
 
 lint_inner_macro_attribute_unstable = inner macro attributes are unstable
 
+lint_invalid_asm_label_binary = avoid using labels containing only the digits 0 and 1 in inline assembly
+    .help = use a different number that has at least one digit 2 or greater
```

Comment:
I'm unsure with the current wording. We mean that it can be *any* value other than 0 or 1, right?
```suggestion
    .label = use number 2 or greater
```

Reply:
`10:` is invalid, which is greater than 2.  A correct label specifically has at least one non-binary digit.  I'm not sure if there's a better way to put this.

Reply:
I see, the issue is even more bizarre than I thought. In that case I'd reword this to

```suggestion
    .help = use a label that doesn't start with `0` or `1`
```

Reply:
It appears that `#[label]` only wants to be applied to a field of type `Span`. Is there a way to tell it to attach to the span that gets emitted with `cx.emit_span_lint(` ?